static/styles/reactions.css

```diff
@@ -53,7 +53,6 @@
         font-size: 90%;
         display: inline-block;
         vertical-align: top;
-        color: hsl(200, 100%, 40%);
```

Comment:
Can you explain this CSS change? It seems like it might affect the display of all reactions.

Reply:
The blue font color made the text look like a link which it is not, therefore this was removed. See this [comment](https://github.com/zulip/zulip/pull/21013#issuecomment-1030442474) for context.

Reply:
OK, in the future please do this sort of "part of the same project but independent idea" change in a separate commit, and provide a commit message explaining the motivation. In this case, you are changing the color for the number in reactions, which it's not clear was intentional, and doing this as a separate change with its own commit message really helps make it possible to review your work.